libs/sdk-backend-spi/src/workspace/users/index.ts

```diff
@@ -58,6 +64,16 @@ export interface IWorkspaceUsersQueryOptions {
      * - basic form match, if it matches as prefix to any of firstName, lastName and email
      */
     search?: string;
+
+    /**
+     * Optionally specify (zero-based) starting offset for the paged results.
+     */
+    offset?: number;
+
+    /**
+     * Optionally specify number of items per page.
+     */
+    limit?: number;
```

Comment:
please specify the what is the default if not provided

Reply:
done